validation/README.md

```diff
@@ -2,3 +2,30 @@
 Tests for reflectivity calculations
 (from Andrew Nelson: https://github.com/andyfaff/orso)
 
+## Polarised tests
+### Coordinate system: Refl1D
+The coordinate system used by Refl1D is based on the book chapter "Polarized Neutron Reflectometry" [^1] by C. F. Majkrzak et al. and is defined as seen in this figure: ![NIST_coords](./NIST_coords.png)
+
+The Sample coordinate system is defined such that the surface normal (parallel to Q) is _z<sub>Sample</sub>_, and then _x<sub>Sample</sub>_  and _y<sub>Sample</sub>_ are in the sample plane.
+
+The Lab coordinate system is defined so that _z<sub>Lab</sub>_ points along the polarisation (guide field = _H_) direction.
+
+Further, _x<sub>Lab</sub>_ is defined to be the same as _x<sub>Sample</sub>_.  Thus _x<sub>Sample</sub>_ is a unit vector in the sample plane (perpendicular to Q) that is also perpendicular to the guide field _H_. _y<sub>Sample</sub>_, _y<sub>Lab</sub>_ are then derived from the right-hand-rule from the _x,z_ components in their respective reference frames, and importantly _H_ is then always in the sample _y,z_ plane.
+
+The angle between _z<sub>Sample</sub>_ and _z<sub>Lab</sub>_ we call _AGUIDE_, and the angle between _x<sub>Sample</sub>_ and the in-plane component of the sample magnetization (_M<sub>IP</sub>_) is _theta<sub>M</sub>_.
```

Comment:
specify the in-plane axes? e.g. (_x<sub>Sample</sub>_y<sub>Sample</sub> plane)?

Reply:
I'm not quite sure what you mean - in the first paragraph I mention that x_sample and y_sample are the in-plane sample axes.  Where else would it make sense to specify them as being in-plane?